data/explorations/all_interactions/all_interactions.yaml

```diff
@@ -1726,3 +1729,4 @@ states:
 states_schema_version: 42
 tags: []
 title: Test of all interactions
+id: '1'
```

Comment:
without adding this id, when I try to load this exploration from the admin page, it throws me an error from the `from_dict` function from the new function I had added. But, the part which I didn't get is other than this demo exploration, only the `audio_tests.yaml` demo exploration works without id, all other exploration throws the error (popup over webpage). 

```
    @classmethod
    def _convert_v54_dict_to_v55_dict(cls, exploration_dict):
        """Converts a v54 exploration dict into a v54 exploration dict.
        Version 55 contains exploration size.

        Args:
            exploration_dict: dict. The dict representation of an exploration
                with schema version v55.

        Returns:
            dict. The dict representation of the Exploration domain object,
            following schema version v55.
        """
        exploration = cls.from_dict(exploration_dict)
        exploration_dict['proto_size_in_bytes'] = (
            exploration.get_proto_size())
        return exploration_dict
```

Error Log
```
Traceback (most recent call last):
  File "opensource/oppia/third_party/python_libs/webapp2.py", line 604, in dispatch
    return method(*args, **kwargs)
  File "opensource/oppia/core/controllers/acl_decorators.py", line 952, in test_super_admin
    return handler(self, **kwargs)
  File "opensource/oppia/core/controllers/admin.py", line 291, in post
    raise e
  File "opensource/oppia/core/controllers/admin.py", line 220, in post
    self._reload_exploration(exploration_id)
  File "opensource/oppia/core/controllers/admin.py", line 307, in _reload_exploration
    exp_services.load_demo(exploration_id)
  File "opensource/oppia/core/domain/exp_services.py", line 1550, in load_demo
    feconf.SYSTEM_COMMITTER_ID, yaml_content, exploration_id, assets_list)
  File "opensource/oppia/core/domain/exp_services.py", line 1486, in save_new_exploration_from_yaml_and_assets
INFO     2021-11-18 12:19:32,207 module.py:865] default: "POST /adminhandler HTTP/1.1" 500 42
    exploration = exp_domain.Exploration.from_yaml(exploration_id, yaml_content)
  File "opensource/oppia/core/domain/exp_domain.py", line 2358, in from_yaml
    exploration_dict = cls._migrate_to_latest_yaml_version(yaml_content)
  File "opensource/oppia/core/domain/exp_domain.py", line 2336, in _migrate_to_latest_yaml_version
    exploration_dict)
  File "opensource/oppia/core/domain/exp_domain.py", line 2253, in _convert_v54_dict_to_v55_dict
    exploration = cls.from_dict(exploration_dict)
  File "opensource/oppia/core/domain/exp_domain.py", line 692, in from_dict
    exploration_dict['id'],
KeyError: 'id'
```

Reply:
Ah, I got it, it's the version of the exploration. 
We can ignore these changes. I will revert this file back.

Reply:
Ahh, need to update the version of the demo exploration so that the failing backend test pass.

Reply:
Fixed by passing the exploration_id to the conversion function.